apps/finance/app/src/lib/icon-utils.js

```diff
@@ -0,0 +1,13 @@
+import { tokenIconUrl as _tokenIconUrl } from '@aragon/ui'
+import { ETHER_TOKEN_VERIFIED_BY_SYMBOL } from './verified-tokens'
+
+// Small shim on top of @aragon/ui's tokenIconUrl, to handle our testnet tokens
+export const tokenIconUrl = (tokenAddress, tokenSymbol, networkType) => {
+  if (networkType === 'main') {
+    return _tokenIconUrl(tokenAddress)
+  }
+
+  // On other networks, pretend any token with the same symbol is legit
+  const mainnetEquivalent = ETHER_TOKEN_VERIFIED_BY_SYMBOL.get(tokenSymbol)
```

Comment:
Couldn’t that be an (minor) issue for people testing an org on rinkeby with a symbol in the list? What about doing it the other way, by having a list of test tokens by network?

Reply:
Yes... we could add an extra check to make sure that the given token address is in the list of test tokens?

In that case, it would **only** be our test tokens that look "legit" with icons displayed in test networks.

Reply:
> we could add an extra check to make sure that the given token address is in the list of test tokens?

Done in 70d65d4.